File: benchmark/comparison/results/adv_py/02.py
def find_first_even(lst):
    for x in lst:
        if x % 2 == 0:
            return x
    return -1

print(find_first_even([1, 3, 2, 5]))
print(find_first_even([1, 3, 5]))
print(find_first_even([4, 2, 6]))

File: benchmark/comparison/results/adv_ts/02.ts
function find_first_even(lst: number[]): number {
  for (const n of lst) {
    if (n % 2 === 0) return n;
  }
  return -1;
}

console.log(find_first_even([1, 3, 2, 5]));
console.log(find_first_even([1, 3, 5]));
console.log(find_first_even([4, 2, 6]));
